server/adaptors/notebooks/index.ts

```diff
@@ -8,10 +8,8 @@ import { DefaultBackend } from './default_backend';
 import { NOTEBOOKS_SELECTED_BACKEND } from '../../../common/constants/notebooks';
 
 // Selects backend based on config
-let BACKEND = new DefaultBackend();
+export let BACKEND: DefaultBackend | ZeppelinBackend = new DefaultBackend();
 
-if (NOTEBOOKS_SELECTED_BACKEND == 'ZEPPELIN') {
+if (NOTEBOOKS_SELECTED_BACKEND === 'ZEPPELIN') {
   BACKEND = new ZeppelinBackend();
 }
-
-export default BACKEND;
```

Comment:
Can we get rid of the Zeppelin backend since it is hard coded to never use it? That will also avoid more confusion in the future

Reply:
Can you create an issue for this, I'll work on it to remove Zeppelin.

Reply:
Created and assigned to you: https://github.com/opensearch-project/dashboards-observability/issues/955. @Swiddis lets not block this PR for this, thanks!